tests/containers/search/filters/naics/mockNAICS.js

```diff
@@ -1,706 +1,806 @@
     {
-        naics: '71',
-        naics_description: 'Arts, Entertainment, and Recreation',
+        value: '71',
+        label: 'Arts, Entertainment, and Recreation',
         count: 5,
         children: [
             {
-                naics: '7111',
-                naics_description: 'Performing Arts Companies',
-                count: 0
-            },
+                isPlaceHolder: true,
+                label: "Placeholder Value",
+                value: "children_of_71"
+            }
+        ]
+    }
+];
+
+export const searchResults = [{
+    value: '11',
+    naics: '11',
+    label: 'Agriculture, Forestry, Fishing and Hunting',
+    count: 64,
+    children: [
+        {
+            value: "1111",
+            naics: "1111",
+            naics_description: "Oilseed and Grain Farming",
+            count: 8,
+            children: [
+                {
+                    value: "111110",
+                    naics: "111110",
+                    naics_description: "Soybean Farming",
+                    count: 0
+                }
+            ]
+        }
+    ]
+}];
+
+export const treeWithPlaceholdersAndRealData = [
+    {
+        value: '11',
+        naics: '11',
+        naics_description: 'Agriculture, Forestry, Fishing and Hunting',
+        count: 64,
+        children: [
             {
-                naics: '7112',
-                naics_description: 'Spectator Sports',
-                count: 3,
+                value: "1111",
+                naics: "1111",
+                naics_description: "Oilseed and Grain Farming",
+                count: 8,
                 children: [
                     {
-                        naics: '711211',
-                        naics_description: 'Sports Teams and Clubs',
+                        value: "111110",
+                        naics: "111110",
+                        naics_description: "Soybean Farming",
                         count: 0
                     },
                     {
-                        naics: '711212',
-                        naics_description: 'Racetracks',
+                        value: "111120",
+                        naics: "111120",
+                        naics_description: "Oilseed (except Soybean) Farming",
                         count: 0
                     },
                     {
-                        naics: '711219',
-                        naics_description: 'Other Spectator Sports',
+                        value: "children_of_1111",
+                        naics: "children_of_1111",
+                        isPlaceHolder: true,
                         count: 0
                     }
                 ]
+            },
+            {
+                value: "children_of_11",
+                naics: "children_of_11",
+                isPlaceHolder: true,
+                count: 0
             }
         ]
     }
 ];
 
-export const naicsMockCleanData = [
+export const treeWithPlaceholdersAndRealDataPSCDepth = [
```

Comment:
Is this for testing NAICS, PSC, or both? Looks like a depth of 4 but NAICS fields. Maybe add a comment to explain.

Reply:
Yeah I added a todo to remove this and replace it with PSC mock data.